src/components/Tag/ListTag.tsx

```diff
@@ -0,0 +1,17 @@
+/** @jsxImportSource theme-ui */
+import React from "react";
+import { styles } from "./styles";
+import { ListTagProps } from "./types";
+import { Flex } from "../Flex";
+
+const ListTag: React.FC<ListTagProps> = ({ variant }) => {
+  return (
+    <Flex variant={variant} sx={styles.listTagCont({ variant })}>
+      <Flex sx={styles.tagText}>
+        {variant} {(variant === "ape" || variant === "uni" || variant === "ark") && "LP"}
```

Comment:
Just make a list of lps and see if its includes so we dont have to add a condition for each